vault/provider.go

```diff
@@ -21,6 +21,10 @@ const (
 	// multiple endpoints in Vault.
 	GenericPath = "generic"
 
+	// MetaPath is used inventorying paths that do not represent any endpoints
+	// in Vault.
+	MetaPath = "meta"
```

Comment:
I wasn't sure what to use here. The data source is not hitting any API endpoint (except for the provider validating the token).